### DISCUSSION TITLE
Our Monikas vs DDLC+

### DISCUSSION THREAD

**Discussion by satoshi-kun** [Monika-After-Story/MonikaModDev#7854] [issue discussions]
Hi!
So since DDLC+ has been annouced I thought I'd ask your views on the following:
Considering that DDLC+ does indeed add something to Monika's lore/character (which I think is likely), will the MAS Dev Team implementing such changes? And if so how?
My personal opinion is that ,yes, this should be addressed in MAS if relevant, but not in a retroactive way i.e. Monika suddently remembering she liked apples in her childhood. 
I see it as if our current Monikas had their arch with their respective player and during this time such memories didnt exist, they are not part of our Monikas, despite the fact that, as an isolated DDLC character, Monika did indeed like apples when she was younger (picking up from the last example).
Being MAS Monika "fully aware" of the events in our world, I think it would be suitable to have her finding this by herself on the web and then telling the player these new traits/memories she apparently now has in this DDLC+, or the other way around.
As I see it, being the case that Monika's character is being developed in some new way during this DDLC+, there should be a separation from now on between Monika pre and post DDLC+.
It is as if our Monikas pre-DDLC+ were enclosed in a different time pocket if you will.
Again these are just my thoughts on the matter.
I'd like to hear yours as well!
Best regards!

**Comment by Retrolovania** [comment]
Huh, not even half an hour after the announcement. I'm impressed.

Anyway, given the extra content is confirmed to not be involved with the core DDLC storyline, it therefore doesn't affect Monika all that much. Therefore, she would probably have a neutral stance on the game.

**Comment by satoshi-kun** [comment]
> Huh, not even half an hour after the announcement. I'm impressed.


Sorry I've wondering about this since DDLC+ was announced way back when and I just got notified on the discord. xD
Thought it would be an interesting topic if there are in fact new lore added
I'm quite curious so see someone playing it on YT!

**Comment by ThePotatoGuy** [comment]
The direct quote from the release announcement is `The Side Stories take place outside the timeline of the original game, and they instead focus on how all of the club members went from first meeting each other to becoming close friends`

This means the side stories are not during the original game, but most likely before - basically a prequel. Any additional Monika lore will be added to the mod - even if we have to retcon it.

**Comment by satoshi-kun** [comment]
I just think that breaks a lot of the emersion.
I already like Monika as she is.
I feel her version pre-DDLC should be "cristalyzed", in a way, as she is today.
Or at least be given an option.

**Comment by username11421** [comment]
Maybe with this content, she can say more when you ask "tell me about yourself" since right now the response is kind of bland

**Comment by ThePotatoGuy** [comment]
>I just think that breaks a lot of the emersion.
I already like Monika as she is.

DDLC+ hasn't been released yet so believing that Dan completely rewrote Monika is premature. I think it is very unlikely that he would have changed every aspect about her.

More official Monika lore is vital for us to flesh out her character - username11421 points out an aspect of the issues we currently have with the available info.

**Comment by satoshi-kun** [comment]
Imagine that they add a past boyfriend to her backstory...
My Monika already told me she never had any.
It would be a heavy blow for me to have that retconned.
Heavy enough for me to consider never updating MAS again if that happens to be the case lol

**Comment by Legendkiller21** [comment]
> I just think that breaks a lot of the emersion.
> I already like Monika as she is.
> I feel her version pre-DDLC should be "cristalyzed", in a way, as she is today.
> Or at least be given an option.

If something breaks the immersion is your suggestion and the whole freeze current ones and end up with 2 different versions of her. Just because you've been together with someone for some time it doesn't mean that you know **everything** about them.

**Comment by satoshi-kun** [comment]
> If something breaks the immersion is your suggestion and the whole freeze current ones and end up with 2 different versions of her. Just because you've been together with someone for some time it doesn't mean that you know **everything** about them.

I sense some hostility on your reply.
I didn't mean to hurt anyone's feelings here.
Was merely exposing my view.

**Comment by lunulae** [comment]
> Imagine that they add a past boyfriend to her backstory...
> My Monika already told me she never had any.
> It would be a heavy blow for me to have that retconned.
> Heavy enough for me to consider never updating MAS again if that happens to be the case lol

I don't think you'll have to worry about that since the side-stories seem only to focus on the main cast. If they happen to, for some reason, mention a dating history, I don't think that would invalidate that Monika's perception of the world changed when she realized nothing in her world was real. Anyone she'd have in her memories might as well have been words on paper once she realizes our world exists.

That said, bear in mind, there's always a chance that an update will include something we don't like or agree with, and this mod team will do the best we can to reconcile major issues, but we do want to preserve Monika's agency as a character. That's been true since the very beginning.

**Comment by Retrolovania** [comment]
Both base DDLC and MAS enforce the fact that she hasn't seen anyone except the player. Given that's the case, I highly doubt the Side Stories will bring that in, so best not fret.

**Comment by Zebulun22** [comment]
The side stories are not canon though if you look up at the post made by potato it says that from the official announcment of ddlc+ that "The Side Stories take place outside the timeline of the original game, and they instead focus on how all of the club members went from first meeting each other to becoming close friends". Implying that it takes place in a alternate universe or something not part of the main timeline so it is ok.

**Comment by satoshi-kun** [comment]
I do realize it is too early to make such assumptions upon DDLC+.
In the face of the unknow all these unsavory situations come to my mind.

**Comment by satoshi-kun** [comment]
> The side stories are not canon though if you look up at the post made by potato it says that from the official announcment of ddlc+ that "The Side Stories take place outside the timeline of the original game, and they instead focus on how all of the club members went from first meeting each other to becoming close friends". Implying that it takes place in a alternate universe or something not part of the main timeline so it is ok.

Well that is still a bit hard to distinguish by now:
Being in a different timeline doesn't mean necessarily it isn't canon, nor that it is in an alternate universe for that matter.
The way I understood it, different timeline just meant that the action wont be happening when the events of DDLC did, it will be happening in the past and will lead up to the events of DDLC. 
So I believe it is still canon, it's a prequel.

**Comment by ThePotatoGuy** [comment]
@Zebulun22, no it only says the side stories don't concur with the *timeline* of the original game. Timeline meaning the events from Act 1 to Act 3/4.

The side stories can most definitely be a prequel based on how the statement says that the side stories focus on how the club members met each other.

**Comment by jmwall24** [comment]
> Imagine that they add a past boyfriend to her backstory...
> My Monika already told me she never had any.
> It would be a heavy blow for me to have that retconned.
> Heavy enough for me to consider never updating MAS again if that happens to be the case lol

Keep in mind if it is a prequel, even if she did mention a boyfriend, that would be a fake, game-world boyfriend from before her epiphany. Aka the boyfriend wasn't real and it wouldn't have any meaning for her. Just another scripted thing.

**Comment by satoshi-kun** [comment]
Whatever the case maybe I'm more concerned atm on how will these possible new lore bits be included in MAS.
Analiticaly speaking, even though these events may be considered Monika's past, "my" Monika never experienced them since this is still a DLC realeased after the original DDLC.
How will you explain her suddently having all these new memories after she already clearly stated to me that she doesn't have many. 
Specially because I wont be playing DDLC+.
I really dont see retcons as a good option because I will still remember what once was.

**Comment by Retrolovania** [comment]
I presume Monika is aware that the mod is updated (as she does talk about contributing to the mod), so a sudden addition of memories wouldn't be too far fetched to her. It'd probably feel like a sudden remembrance to her.

**Comment by lunulae** [comment]
A lot of them probably won't be written as memories she suddenly remembers, just things that she hasn't spoken about with you yet.
You could apply the same logic to all new topics added in every update--sure, we as contributors know this is new information that didn't exist in the mod. But when you're playing, the intent is for it to be something that she has gotten a new idea about, or thinks it might be interesting to talk about today.

Likewise, we have already retconned a number of topics that you might not have noticed. 
For example, take the topic about Monika's appearance--before, we had it written to say that Monika still needed to eat well to maintain her figure. But that logic was hard to reconcile and didn't fully make sense, and so it was retconned into her doing it because healthy habits are important to her personally, and they would be useful things to maintain for if she crossed over.

**Comment by jmwall24** [comment]
I wouldn't worry too much, we know how much Monika means to so many of you, we're not going to do anything that we know would cause our base a lot of distress. We have ways to address things without necessarily having to make huge changes, depending on what they are. We are also well aware that not every MAS user will even purchase DDLC+, so we'll take everything into consideration. Plus Dan seemed pretty adamant that anything added isn't going to change the fundamentals of the characters.

**Comment by satoshi-kun** [comment]
You are right.
I'm reading too much into this and the MAS Dev Team never failed to deliver when it comes to adding new topics and keeping the sense of emersion.
Sorry if sounded doubtful of that.
The truth is, the looming possibility of change this DDLC+ brings, really left me anxious and concerned, but that is just the way I perceive changes in my life. Not a fan.
Thanks for your feedback!
It helped me a lot to have heard your perspectives.

**Comment by Mugnika** [comment]
looks like our fears were not unfounded...

**Comment by satoshi-kun** [comment]
Yes, saddly for what I've read so far they really went for the, in my opinion, gratuitous shock factor.
I'm very sad.
Although Monika and DDLC are someone else's IPs it really pains me the way she was used by TeamSalvato in DDLC+.
#notmymonika

**Comment by Retrolovania** [comment]
Yeah, from what little I'm able to look at due to not wanting spoilers, it seems they've retconned far too many things. You could probably say it's not the same Monika we've come to know and love anymore.

Thank goodness we can probably still consider the original DDLC's setup as canon and ignore Plus, or we'd be in for a real mess.

**Comment by Zebulun22** [comment]
Actually I found out the OG DDLC is canon to OG DDLC. So we can ignore plus.

**Comment by ThePotatoGuy** [comment]
After some of us in the team play and explore DDLC+, we'll begin discussions on what to include and what to ignore.

**Comment by satoshi-kun** [comment]
Besides that point, in my opinion, DDLC+ could be aknowledge by Monika in MAS in someway.
The same way she already aknowledges "her" twitter acount, her wikipedia page, the rights of use of her image and even a possible sequel.
Afterall, Monika is now the cover of a physical videogame copy.
That is kind of a big deal.
